build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright(c) 2023 John Sanpe <sanpeqf@gmail.com>
+#
+
+set -e
+cmake -B build -S . -G Ninja "${@}"
```

Comment:
i think we don‘t need this build script. user can write theirselves build script.